[2025-06-19T09:08:46.244+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-19T09:08:46.273+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kse100_forecasting.train_stacked manual__2025-06-19T09:08:22.823079+00:00 [queued]>
[2025-06-19T09:08:46.281+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kse100_forecasting.train_stacked manual__2025-06-19T09:08:22.823079+00:00 [queued]>
[2025-06-19T09:08:46.282+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-06-19T09:08:46.297+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): train_stacked> on 2025-06-19 09:08:22.823079+00:00
[2025-06-19T09:08:46.304+0000] {standard_task_runner.py:63} INFO - Started process 692 to run task
[2025-06-19T09:08:46.307+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'kse100_forecasting', 'train_stacked', 'manual__2025-06-19T09:08:22.823079+00:00', '--job-id', '1103', '--raw', '--subdir', 'DAGS_FOLDER/kse100_forecasting_dag.py', '--cfg-path', '/tmp/tmpkoldairc']
[2025-06-19T09:08:46.310+0000] {standard_task_runner.py:91} INFO - Job 1103: Subtask train_stacked
[2025-06-19T09:08:46.351+0000] {task_command.py:426} INFO - Running <TaskInstance: kse100_forecasting.train_stacked manual__2025-06-19T09:08:22.823079+00:00 [running]> on host fb048d2dbcb2
[2025-06-19T09:08:46.465+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kse100_forecasting' AIRFLOW_CTX_TASK_ID='train_stacked' AIRFLOW_CTX_EXECUTION_DATE='2025-06-19T09:08:22.823079+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-19T09:08:22.823079+00:00'
[2025-06-19T09:08:46.466+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-19T09:08:49.898+0000] {logging_mixin.py:188} INFO - Stacked: R2=0.3407, MSE=16635271.0878, MAE=3313.1310
[2025-06-19T09:08:49.903+0000] {logging_mixin.py:188} INFO - Error predicting on full dataset (Stacked): [Errno 2] No such file or directory: '/opt/***/data/engineered_features.csv'
[2025-06-19T09:08:49.904+0000] {python.py:237} INFO - Done. Returned value was: ('/opt/***/dataKse/models/kse_stacked_model.pkl', '/opt/***/dataKse/models/kse_stacked_metrics.csv')
[2025-06-19T09:08:49.904+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-19T09:08:49.933+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=kse100_forecasting, task_id=train_stacked, run_id=manual__2025-06-19T09:08:22.823079+00:00, execution_date=20250619T090822, start_date=20250619T090846, end_date=20250619T090849
[2025-06-19T09:08:49.970+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-19T09:08:49.985+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-19T09:08:49.990+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
